geometry_msgs/PoseStamped grasp_pose_stamped
---
float64 affordable_distance